client/main.js

```diff
@@ -0,0 +1,470 @@
+import BN from "bn.js";
```

Comment:
I am kind of reluctant to the idea to put this client into this repository. Do we think it's possible to extract it somehow? We built means for that in the colony-js client. Maybe we can have a discussion around that.

Reply:
Also pinging @JamesLefrere here.